vignettes/cmdstanr.Rmd

```diff
@@ -158,39 +162,52 @@ fit <- mod$sample(
 )
 ```
 
-The
-[`$sample()`](https://mc-stan.org/cmdstanr/reference/model-method-sample.html)
-method creates [R6](https://r6.r-lib.org/)
-[`CmdStanMCMC`](https://mc-stan.org/cmdstanr/reference/CmdStanMCMC.html)
-objects.
+There are many more arguments that can be passed to the `$sample()` method. 
+For details follow this link to its separate documentation page: 
+
+* [`$sample()`](https://mc-stan.org/cmdstanr/reference/model-method-sample.html)
+
+The `$sample()` method creates [R6](https://r6.r-lib.org/) `CmdStanMCMC`
+objects, which have many associated methods. Below we will demonstrate some
+of the most important methods. For a full list, follow this link to the `CmdStanMCMC` documentation: 
+
+* [`CmdStanMCMC`](https://mc-stan.org/cmdstanr/reference/CmdStanMCMC.html)
 
 #### Posterior summary statistics
 
-The [`$summary()`](https://mc-stan.org/cmdstanr/reference/fit-method-summary.html)
-method calls `summarise_draws()` from the **posterior** package: 
+The
+[`$summary()`](https://mc-stan.org/cmdstanr/reference/fit-method-summary.html)
+method calls `summarise_draws()` from the **posterior** package:
```

Comment:
Link to https://mc-stan.org/posterior/reference/draws_summary.html or https://mc-stan.org/posterior/index.html here maybe?

Reply:
Yeah, that link should get automatically created by [pkgdown](https://pkgdown.r-lib.org/) when we generate the website. You can see that it's hyperlinked here:

http://mc-stan.org/cmdstanr/articles/cmdstanr.html#posterior-summary-statistics

The reason we have to manually do the links for, e.g., `$summary()` is because pkgdown doesn't understand that we would want it to link to the summary method for CmdStanMCMC objects (it would just think we want a link to base::summary).

Reply:
Ah, ok :) Good to know when I start working on vignettes.